### ISSUE TITLE
Add pixie.js wherever there are animations, moving etc. 

### ISSUE THREAD

**Issue by MadejaMaciej** [niner-games/game-react-lost-dutchman-mine#55]
It is efficient game like engine for js.

**Comment by mrfatguy** [comment]
Make sure that you perform extensive performance tests once you add this and compile this library into our app!

As a "webapp" ([pixijs.com](https://pixijs.com/examples/events/custom-mouse-icon) website opened directly) it seems awfully slow. Each and [every example](https://pixijs.com/examples/events/custom-mouse-icon) pulses "P" (loader) for at least 3-4 seconds before actually showing some render. And PC's fan gets crazy even though its _13th Gen Intel i7-1355U + 32 GB_ here!

Task Manager in Windows 11 shows **CPU consumption boosting up to 15-20%** from 0.2-0.5% when only switching from GitHub to their website opened in another tab. And to **even up to 40-50%** when rendering some new scene / example.

I don't know how about you, but I'd call that a terrible performance! :)

**Comment by mrfatguy** [comment]
The [three.js](https://threejs.org/examples/#webgl_animation_keyframes) and [pannellum.org](https://pannellum.org/) gets roughly 10-15% with a very rare situations when getting picks up to 20%. Rendering is immediate, without any loader or waiting. But these are both 3D libraries, so I don't know, if you can adapt them into 2D game.

There's Box2D library ([blog post](http://buildnewgames.com/2d-platformer-character-movement/), [library](https://box2d.org/about/)) for 2D physics, but doesn't have any on-line examples, so I can't check performance.

And there's [this 10+ years old article](https://archive.jlongster.com/Making-Sprite-based-Games-with-Canvas), if you'd like to consider crafting own solution based on canvas.

**Comment by MadejaMaciej** [comment]
Okay, that scraps it up basically :D We do not want that for sure.

I'll write own animations engine :)

**Comment by MadejaMaciej** [comment]
Yeah, that was terrible idea -> react alone will probably will quicker. If performance for React will be poor I then use special library for that.
